packages/grid/x-data-grid-generator/package.json

```diff
@@ -44,8 +44,8 @@
     "yargs": "^17.5.1"
   },
   "peerDependencies": {
-    "@mui/icons-material": "^5.2.5",
-    "@mui/material": "^5.2.8",
+    "@mui/icons-material": "^5.4.1",
+    "@mui/material": "^5.4.1",
```

Comment:
This might be considered a breaking change, but I would argue that the breaking change was already released in https://github.com/mui/mui-x/issues/2998#issuecomment-1036337411 so we might as well be upfront with it.